app/Http/Controllers/AuthController.php

```diff
@@ -2,29 +2,29 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
+use App\Http\Requests\UserRequest;
+use App\Http\Resources\UserResource;
+use Illuminate\Http\JsonResponse;
 use App\Models\User;
+use Illuminate\Http\Request;
 
 class AuthController extends Controller
 {
-    public function register(Request $request)
+    public function register(UserRequest $request): JsonResponse
     {
-        $validated = $request->validate([
-            'name' => 'required|string',
-            'email' => 'required|email|unique:users,email',
-            'password' => 'required|string|min:6',
-        ]);
-
-//        dd ($validated);
-
+        $user = User::create($request->validated());
+        return (new UserResource($user))->response()->setStatusCode(201);
```

Comment:
or
```suggestion
        return (new UserResource(User::create($request->validated())))
        ->response()
        ->setStatusCode(201);
```
(and add tabulation)